dts/arm/st/h7/stm32h723.dtsi

```diff
@@ -234,6 +234,7 @@
 	otghs_fs_phy: otghs_fs_phy {
 		compatible = "usb-nop-xceiv";
 		#phy-cells = <0>;
+		clocks = <&rcc STM32_CLOCK(AHB2, 31U)>;
```

Comment:
Could you crosscheck that? It's seems to me USB1OTGHSULPIEN is located in RCC_AHB1xxR[bit 26] on stm32h723/h725.